tests/Vue/Pages/Home.spec.js

```diff
@@ -18,21 +18,17 @@ describe('Home.vue', () => {
     const localVue = createLocalVue();
     localVue.use(Vuex);
 
-    const store = new Vuex.Store();
-
     it('sanitises input with empty lines', async () => {
-        const formData = { itemsInput: 'Q1\n\nQ2\n' };
+
+        const store = new Vuex.Store({state: { lastSearchedIds: 'Q1\n\nQ2\n' }});
```

Comment:
Ah, here the unconditional replacement of the `form.itemsInput` bites you, right? :-)

I think the two features should not be mixed in a single test. One should be testing that input is sanitised and a second one should make sure that the input is restored when `lastSearchedIds` is passed via the Store. In the first case, when mounting the wrapper with an empty store, I guess you will now have to set `form.itemsInput` directly.

Reply:
Two tests added. Good observation.